lib/syscall_shim/uk_prsyscall.c

```diff
@@ -1003,6 +1046,25 @@ static void pr_syscall(struct uk_streambuf *sb, int fmtf,
 
 #endif /* HAVE_uk_syscall_socket */
 
+#ifdef HAVE_uk_syscall_clone
+	case SYS_clone:
+#if CONFIG_ARCH_X86_64
+		VPR_SYSCALL(sb, fmtf, syscall_num, args, PT_CLONEFLAGS,
+			    PT_VADDR, /* sp */
+			    PT_TID | PT_REF, /* ref to parent tid */
+			    PT_TID | PT_REF, /* ref to child tid */
+			    PT_VADDR /* tlsp */ );
+#else /* !CONFIG_ARCH_X86_64 */
+		VPR_SYSCALL(sb, fmtf, syscall_num, args, PT_CLONEFLAGS,
+			    PT_VADDR, /* sp */
+			    PT_TID | PT_REF, /* ref to parent tid */
+			    PT_VADDR, /* tlsp */
+			    PT_TID | PT_REF /* ref to child tid */ );
+#endif /* !CONFIG_ARCH_X86_64 */
```

Comment:
It seems "Checkpatch-Ignore: SPACING" is needed to keep the inline comments.

```
ERROR:SPACING: space prohibited before that close parenthesis ')'
#94: FILE: lib/syscall_shim/uk_prsyscall.c:1056:
+			    PT_VADDR /* tlsp */ );

ERROR:SPACING: space prohibited before that close parenthesis ')'
#100: FILE: lib/syscall_shim/uk_prsyscall.c:1062:
+			    PT_TID | PT_REF /* ref to child tid */ );

total: 2 errors, 0 warnings, 86 lines checked
```

Other than that, looks good to me. Thank you.

Reply:
:+1: I'll update it